paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButtonNew.kt

```diff
@@ -73,7 +72,7 @@ private const val CENTER_ALIGNED = 0f
 internal sealed interface PrimaryButtonProcessingState {
     data class Idle(val error: ResolvableString?) : PrimaryButtonProcessingState
 
-    object Processing : PrimaryButtonProcessingState
+    data class Processing(val subdueText: Boolean = true) : PrimaryButtonProcessingState
```

Comment:
I added this as part of the state but I'm still mixed on if it should be a part of the state or just a `PrimaryButton` parameter with a default value

Reply:
When would we have a processing state without subdued text?

Reply:
The base case in `PaymentSheet` currently has `Processing...` as not subdued while saving in `CustomerSheet` does.

Reply:
Let’s align the two. @jarvis-stripe, what do you think?

Reply:
I think when a primary CTA show "processing..." it should be readable, so let's not subdue it

Reply:
In the case of `CustomerSheet`, the `Save` label is shown & subdued when processing. It doesn't use `Processing...`. Should we update `CustomerSheet` to use `Processing...` instead?